src/constructs/autoscaling/user-data.ts

```diff
@@ -11,7 +12,6 @@ import type { AppIdentity } from "../core/identity";
  * `executionStatement` will be something like "dpkg -i application.deb` or `service foo start`.
  */
 export interface GuUserDataS3DistributableProps {
-  bucket: GuDistributionBucketParameter;
   fileName: string;
```

Comment:
It's probably worth mentioning where the bucket name is coming from in this comment (i.e. something similar to [this one](https://github.com/guardian/cdk/blob/2692d040c70486fbe5acb11075e4fc0a027d86fa/src/constructs/lambda/lambda.ts#L23-L25)). Tangential, but this makes me wonder if we should fix the [`key`](https://github.com/guardian/cdk/blob/2692d040c70486fbe5acb11075e4fc0a027d86fa/src/constructs/lambda/lambda.ts#L23-L26)/`fileName` inconsistency too?

Reply:
> It's probably worth mentioning where the bucket name is coming from in this comment

👍🏽  thanks for the reminder!

> Tangential, but this makes me wonder if we should fix the [`key`](https://github.com/guardian/cdk/blob/2692d040c70486fbe5acb11075e4fc0a027d86fa/src/constructs/lambda/lambda.ts#L23-L26)/`fileName` inconsistency too?

Absolutely! Spoiler alert: this is going to be raised in a couple of PRs 😄 .

Reply:
I thought you'd probably notice it too! 😁